packages/program-boilerplate/src/jsonata.ts

```diff
@@ -64,3 +65,25 @@ export function safeJsonata(expression: string, inputData: any) {
     logger.warn(`Failed to evaluate JSONata expression: ${e.message}`);
   }
 }
+
+export function safeJsonata2(
```

Comment:
I would prefer if `safeJsonata` just got new optional arguments with defaults, rather than `safeJsonata2`.

Reply:
Please see my comment earlier in this PR. The problem is not with the arguments, it's with the return type signature.

Reply:
Thanks @jayden-chan, I missed your earlier comment. Could we at least rename the function to something better than `safeJsonata2`?